rwengine/src/audio/SoundManager.cpp

```diff
@@ -16,6 +18,9 @@ extern "C" {
 #define av_frame_free   avcodec_free_frame
 #endif
 
+constexpr int kNrOfOutputChannels = 2;
```

Comment:
nit: At first look I didn't understand what was the purpose of this, so `kNumOutputChannels` maybe?